ods_ci/tests/Resources/Files/accelerator/qe_create_ap_3.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: dashboard.opendatahub.io/v1
+kind: AcceleratorProfile
+metadata:
+  name: qecreateap3
+  namespace: redhat-ods-applications
+spec:
+  description: description example
+  displayName: qe_create_ap_3
```

Comment:
Can you change description and displayName with a meaningful name here as well ?